### PR TITLE
Fixed: Various Translations

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Todos
- [x] Tests
- [x] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [x] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

File: frontend/src/System/Status/Donations/Donations.js [hunk 1]
```diff
@@ -12,39 +12,39 @@ class Donations extends Component {
   render() {
     return (
       <FieldSet legend={translate('Donations')}>
-        <div className={styles.logoContainer} title="Radarr">
+        <div className={styles.logoContainer} title={translate('Radarr')}>
```
Comment (Qstick) [inline frontend/src/System/Status/Donations/Donations.js:15]: We don't actually want people to be able to change these via translation. Thus we haven't wrapped them.

File: frontend/src/Settings/Quality/Definition/QualityDefinitions.js [hunk 2]
```diff
@@ -25,9 +25,15 @@ class QualityDefinitions extends Component {
...
-            <div className={styles.title}>Title</div>
-            <div className={styles.sizeLimit}>Size Limit</div>
+            <div className={styles.quality}>
+              {translate('Quality')}
+            </div>
+            <div className={styles.title}>
+              {translate('Title')}
+            </div>
+            <div className={styles.sizeLimit}>
+              {translate('SizeLimit')}
```
Comment (Qstick) [inline frontend/src/Settings/Quality/Definition/QualityDefinitions.js:35]: Not in en.json

File: frontend/src/Settings/DownloadClients/RemotePathMappings/RemotePathMappings.js [hunk 3]
```diff
@@ -51,9 +51,15 @@ class RemotePathMappings extends Component {
...
-            <div className={styles.path}>Remote Path</div>
-            <div className={styles.path}>Local Path</div>
+            <div className={styles.host}>
+              {translate('Host')}
+            </div>
+            <div className={styles.path}>
+              {translate('RemotePath')}
+            </div>
+            <div className={styles.path}>
+              {translate('LocalPath')}
```
Comment (Qstick) [inline frontend/src/Settings/DownloadClients/RemotePathMappings/RemotePathMappings.js:61]: Not in en.json

File: frontend/src/Settings/DownloadClients/RemotePathMappings/RemotePathMappings.js [hunk 4]
```diff
@@ -51,9 +51,15 @@ class RemotePathMappings extends Component {
...
         >
           <div className={styles.remotePathMappingsHeader}>
-            <div className={styles.host}>Host</div>
-            <div className={styles.path}>Remote Path</div>
-            <div className={styles.path}>Local Path</div>
+            <div className={styles.host}>
+              {translate('Host')}
+            </div>
+            <div className={styles.path}>
+              {translate('RemotePath')}
```
Comment (Qstick) [inline frontend/src/Settings/DownloadClients/RemotePathMappings/RemotePathMappings.js:58]: Not in en.json

**Review by Qstick — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
fixed up @austinwbest mental note: language script, need to review why it didnt add to en.json & why missing did not capture those

---

q - diffs for the latest force push
image
image
image

File: frontend/src/InteractiveImport/Language/SelectLanguageModalContent.js [hunk 5]
```diff
@@ -128,7 +128,7 @@ class SelectLanguageModalContent extends Component {
             kind={kinds.SUCCESS}
             onPress={this.onLanguageSelect}
           >
-            {translate('SelectLanguges')}
+            {translate('SelectLanguage')}
```
Comment (Qstick) [inline frontend/src/InteractiveImport/Language/SelectLanguageModalContent.js:131]: SelectLanguages

File: frontend/src/Settings/ImportLists/ImportListExclusions/ImportListExclusions.js [hunk 6]
```diff
@@ -50,10 +50,16 @@ class ImportListExclusions extends Component {
           errorMessage={translate('UnableToLoadListExclusions')}
           {...otherProps}
         >
-          <div className={styles.importListExclusionsHeader}>
-            <div className={styles.tmdbId}>TMDB Id</div>
-            <div className={styles.title}>Title</div>
-            <div className={styles.movieYear}>Year</div>
+          <div className={styles.importExclusionsHeader}>
+            <div className={styles.tmdbId}>
+              {translate('TMDBId')}
```
Comment (Qstick) [inline frontend/src/Settings/ImportLists/ImportListExclusions/ImportListExclusions.js:55]: Don't translate

**Review by Qstick — CHANGES_REQUESTED** [review]

**Review by Qstick — APPROVED** [review]

**Pull request merged**